docs/contents/components/typography/text/index.ja.mdx

```diff
@@ -26,45 +26,13 @@ import { Text } from "@yamada-ui/react"
 
 ```tsx
 <VStack>
-  <Text fontSize="6xl" isTruncated>
-    私の戦闘力は530000です。ですがもちろんフルパワーであなたと戦う気はありませんからご心配なく……
-  </Text>
-
-  <Text fontSize="5xl" isTruncated>
-    私の戦闘力は530000です。ですがもちろんフルパワーであなたと戦う気はありませんからご心配なく……
-  </Text>
-
-  <Text fontSize="4xl" isTruncated>
-    私の戦闘力は530000です。ですがもちろんフルパワーであなたと戦う気はありませんからご心配なく……
-  </Text>
-
-  <Text fontSize="3xl" isTruncated>
-    私の戦闘力は530000です。ですがもちろんフルパワーであなたと戦う気はありませんからご心配なく……
-  </Text>
-
-  <Text fontSize="2xl" isTruncated>
-    私の戦闘力は530000です。ですがもちろんフルパワーであなたと戦う気はありませんからご心配なく……
-  </Text>
-
-  <Text fontSize="xl" isTruncated>
-    私の戦闘力は530000です。ですがもちろんフルパワーであなたと戦う気はありませんからご心配なく……
-  </Text>
-
-  <Text fontSize="lg" isTruncated>
-    私の戦闘力は530000です。ですがもちろんフルパワーであなたと戦う気はありませんからご心配なく……
-  </Text>
-
-  <Text fontSize="md" isTruncated>
-    私の戦闘力は530000です。ですがもちろんフルパワーであなたと戦う気はありませんからご心配なく……
-  </Text>
-
-  <Text fontSize="sm" isTruncated>
-    私の戦闘力は530000です。ですがもちろんフルパワーであなたと戦う気はありませんからご心配なく……
-  </Text>
-
-  <Text fontSize="xs" isTruncated>
-    私の戦闘力は530000です。ですがもちろんフルパワーであなたと戦う気はありませんからご心配なく……
-  </Text>
+  <For each={["6xl", "5xl", "4xl", "3xl", "2xl", "xl", "lg", "md", "sm", "xs"]}>
+    {(fontSize, key) => (
+      <Text key={key} fontSize={fontSize} isTruncated>
+        私の戦闘力は530000です。ですがもちろんフルパワーであなたと戦う気はありませんからご心配なく……
+      </Text>
+    )}
```

Comment:
In different documents, it's called '`index`' instead of '`key`.' Please adjust this part.

```suggestion
    {(fontSize, index) => (
      <Text key={index} fontSize={fontSize} isTruncated>
        私の戦闘力は530000です。ですがもちろんフルパワーであなたと戦う気はありませんからご心配なく……
      </Text>
    )}
```